chapter_recurrent-neural-networks/language-models-and-dataset_vn.md

```diff
@@ -39,7 +46,11 @@ Likewise, in a document summarization algorithm it is worth while knowing that "
 or that "I want to eat grandma" is a rather disturbing statement, whereas "I want to eat, grandma" is much more benign.
 -->
 
-*dịch đoạn phía trên*
+Tuy nhiên, những mô hình ngôn ngữ như thế này vẫn rất hữu dụng ngay cả khi chúng vẫn còn những mặt hạn chế.
+Chẳng hạn, cụm từ  “to recognize speech” và “to wreck a nice beach” khi nghe phát âm thì có vẻ rất giống nhau.
```

Comment:
```suggestion
Chẳng hạn, cụm từ  “để nhận dạng giọng nói” và “to nhân gian giông tố" khi nghe phát âm thì có vẻ rất giống nhau.
```
mình biến tấu cụm này sang tiếng việt luôn

Reply:
chỗ này `“to nhân gian giông tố”` hơi kỳ...

Reply:
ý mình ko có chữ `to`, đã sửa.
Còn cái `giống tố` chỉ bịa đại ra thôi à :)).